### PR TITLE
Updates after repo split

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
Adapt repo after split - remove elrond-go dependency

### CODE REVIEW

File: errors.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package storage
+package elrond_go_storage
```
Comment (sstanculeanu) [inline errors.go]: perhaps move this file into `common/errors.go` or rename the package

File: interface.go [hunk 2]
```diff
@@ -1,11 +1,9 @@
-package storage
+package elrond_go_storage
```
Comment (sstanculeanu) [inline interface.go]: maybe move this file as well into common?

File: errors.go (hunk 1 above)
Comment (bogdan-rosianu) [reply errors.go]: moved to a new package

File: interface.go (hunk 2 above)
Comment (bogdan-rosianu) [reply interface.go]: moved to a new package

**Review by AdoAdoAdo — CHANGES_REQUESTED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

File: .github/workflows/pr-tests.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+jobs:
+  test:
+    name: Unit
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.15.6
```
Comment (AdoAdoAdo) [inline .github/workflows/pr-tests.yml]: range?

Comment (bogdan-rosianu) [reply .github/workflows/pr-tests.yml]: added go 1.17.6

**Review by sstanculeanu — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Pull request merged**